src/Http/ActionDispatcher.php

```diff
@@ -47,7 +47,7 @@ public function __construct(?ControllerFactory $factory = null)
      * Dispatches a Request & Response
      *
      * @param \Cake\Http\ServerRequest $request The request to dispatch.
-     * @param \Psr\Http\Message\ResponseInterface $response The response to dispatch.
+     * @param \Cake\Http\Response $response The response to dispatch.
```

Comment:
I don't think the docblock not matching the actual typehint is a good idea. Inline `@var` can be added inside the method where required to make phpstan happy.